yarn.lock

```diff
@@ -27022,6 +27022,11 @@ prettier@^2.5.1, prettier@^2.7.1:
   resolved "https://registry.yarnpkg.com/prettier/-/prettier-2.7.1.tgz#e235806850d057f97bb08368a4f7d899f7760c64"
   integrity sha512-ujppO+MkdPqoVINuDFDRLClm7D78qbDt0/NR+wp5FqEZOoTNAjPHWj17QRhu7geIHJfcNhRk1XVQmF8Bp3ye+g==
 
+prettier@^2.8.7:
```

Comment:
With this one, we have 4 different prettier versions installed. One of them is restricted by storybook (maybe we should update it too soon), but the others are all in the same range, so they can be unified into one.

Manually deleting the entries from yarn.lock and running `yarn` again will solve this.

Reply:
We should not manually edit yarn.lock, but i will run it again to regenerate

Reply:
they are still might be installed because they specified as pear dependencies somewhere, not only storybook needs it, flowgen needs 2.7.1. I won't manually remove them, cause running it again will add them again, i removed it inside our babel-plugin as dev dep instead, to remove 2.2.1

Reply:
> cause running it again will add them again

No, it won't.
Flowgen needs `^2.5.1` and is resolving to 2.7.1 but could resolve to 2.8.7 like all others and avoid 2.7.1 to be installed entirely.
Even if they are peer deps, version 2.8.7 satisfies all the requirements. Only version 2.3.0 will be installed, because it's the maximum limit version required by a storybook package.
VSCode disallowing you should be some setting on your side, because I can delete entries on yarn.lock. But if you don't want to do it, we can actually use [a tool](https://www.npmjs.com/package/yarn-deduplicate) for it.

Reply:
It's not a problem, I will just use nvim, let's see